server/routes/routes.go

```diff
@@ -29,11 +30,11 @@ type SystemInfo struct {
 	Namespaced       bool   `json:"namespaced"`
 }
 
-func Routes(r *gin.Engine, sysinfo SystemInfo) {
+func Routes(r *gin.Engine, sysinfo SystemInfo, baseHref string) {
 	r.GET("/livez", func(c *gin.Context) {
 		c.Status(http.StatusOK)
 	})
-	rGroup := r.Group("/api/v1")
+	rGroup := r.Group(path.Join(baseHref, "api/v1"))
```

Comment:
When `baseHref` is empty string, is that a problem?

Reply:
No, when string is empty these are the paths:

```
[GIN-debug] GET    /livez                    --> github.com/numaproj/numaflow/server/routes.Routes.func1 (3 handlers)
[GIN-debug] GET    /api/v1/namespaces/:namespace/pipelines --> github.com/numaproj/numaflow/server/apis/v1.(*handler).ListPipelines-fm (3 handlers)
[GIN-debug] GET    /api/v1/namespaces/:namespace/isbsvcs --> github.com/numaproj/numaflow/server/apis/v1.(*handler).ListInterStepBufferServices-fm (3 handlers)
```

Reply:
When `baseHref` is an empty string, wouldn't the joined string be `api/v1`?